src/Services/RequestHelpers.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 import os from 'os';
+import { Agent } from 'http';
+const HttpsProxyAgent = require('https-proxy-agent');
```

Comment:
Hmmm, can this not be imported like `import {HttpsProxyAgent} from 'https-proxy-agent';`?

Reply:
@DarthHater the code has been changed as requested.
I made a fixup. If this version is agreed, I'll squash it to have a single commit